luigi_pipeline/seqr_loading.py

```diff
@@ -130,4 +130,5 @@ def run(self):
 
 
 if __name__ == '__main__':
-    luigi.run()
+    # If run does not succeed, exit with 1 status code.
+    luigi.run() or exit(1)
```

Comment:
I think `sys.exit` would be preferable here.

https://docs.python.org/3/library/constants.html#constants-added-by-the-site-module

Reply:
Ahh yes I had `from sys import exit` but it was removed when I reset the branch and forgot to add it. Good catch.